repos/system_upgrade/common/actors/checksaphana/actor.py

```diff
@@ -12,17 +14,52 @@ class CheckSapHana(Actor):
     If the upgrade flavour is 'default' no checks are being executed.
 
     The following checks are executed:
-    - If this system is _NOT_ running on x86_64, the upgrade is inhibited.
-    - If SAP HANA 1 has been detected on the system the upgrade is inhibited since it is not supported on RHEL8.
+    - If the major target release is 8, and this system is _NOT_ running on x86_64, the upgrade is inhibited.
+    - If the major target release is 9, and this system is _NOT_ running on x86_64 or ppc64le,
+      the upgrade is inhibited.
+    - If SAP HANA 1 has been detected on the system the upgrade is inhibited since there is no supported upgrade path
+      with installed SAP HANA 1.
     - If SAP HANA 2 has been detected, the upgrade will be inhibited if an unsupported version for the target release
-      has been detected.
+      has been detected (<8.8, <9.2).
+    - If the target release >=8.8 or >=9.2, the upgrade will be inhibited unless a user confirms to proceed
+      for the currently installed SAP HANA 2.0 version and the chosen target release.
     - If SAP HANA is running the upgrade is inhibited.
     """
 
     name = 'check_sap_hana'
     consumes = (SapHanaInfo,)
     produces = (Report,)
     tags = (IPUWorkflowTag, ChecksPhaseTag)
+    dialogs = (
+        Dialog(
+            scope='check_sap_hana',
+            reason='Confirmation',
+            components=(
+                BooleanComponent(
+                    key='confirm',
+                    label='Do you want the upgrade to proceed for '
+                          'the currently installed SAP HANA 2.0 version?',
+                    description='Enter True, otherwise the upgrade process '
```

Comment:
One should not write 'True' but 'Yes'

https://github.com/oamg/leapp/blob/9eb9c463fc849005378a34aac22a44a9cb1a662c/leapp/dialogs/components.py#L88

Reply:
I will correct this one, as soon we agree on other comments @vinzenz Thanks!

Reply:
@vinzenz @alexxa the `True` could works also (we use it in other place already), but yes seems to be more natural answer to the question, so let's stay with `Yes`

Reply:
I suggest to set also the default, so the cmd will be generated in the report that could be executed by user